xarray/backends/zarr.py

```diff
@@ -588,7 +613,7 @@ def open_zarr(
         values according to the formula `original_values * scale_factor +
         add_offset`, where `_FillValue`, `scale_factor` and `add_offset` are
         taken from variable attributes (if they exist).  If the `_FillValue` or
-        `missing_value` attribute contains multiple values a warning will be
+            `missing_value` attribute contains multiple values a warning will be
```

Comment:
the errors are:
```
/home/docs/checkouts/readthedocs.org/user_builds/xray/checkouts/5065/xarray/backends/zarr.py:docstring of xarray.backends.zarr.open_zarr:28: WARNING: Unexpected indentation.
/home/docs/checkouts/readthedocs.org/user_builds/xray/checkouts/5065/xarray/backends/zarr.py:docstring of xarray.backends.zarr.open_zarr:29: WARNING: Block quote ends without a blank line; unexpected unindent.
```

my guess is that both would be fixed by undoing this:
```suggestion
        `missing_value` attribute contains multiple values a warning will be
```

Reply:
not sure how, but this seems to have been reintroduced and causes the docs build to fail.